taxcalc/parameters.py

```diff
@@ -6,7 +6,7 @@
 
 DEFAULT_START_YEAR = 2013
 
-class Parameters(object):
+class Params(object):
```

Comment:
I thought that it might be confusing to API users if Calculator attribute and class name match for Records and records, but not for Parameters and params.

The question is whether this is confusing: 

    tax_dta = pd.read_csv("../../puf.csv")
    # Create a default Parameters object
    paramsX = Parameters(start_year=2013)
    recordsX = Records(tax_dta)
    # Create a Calculator
    calcX = Calculator(records=recordsX, params=paramsX)

 I do agree that Parameters is a better name for the class, though. What do you think?